### ISSUE TITLE
Requesting support for .vos (in addition to .vio) files

### ISSUE THREAD

**Issue by yoichi-at-bedrock** [cpitclaudel/company-coq#260]
Currently, `M-.` on a library name doesn't work for libraries found in `.vos` files (it works when the library is found in a `.vo` file).

Perhaps `company-coq-compiled-regexp` can be modified to allow `.vos` suffix?

**Comment by cpitclaudel** [comment]
Indeed, it should be relatively easy to add support for `vos`. I'm old school, so I haven't actually used them, but I'd be happy to merge a PR.
